### PR TITLE
Getting Started - minor fixes

### PR DESCRIPTION
**Pull request by bmegli**
BrickPi sentence "With BrickPi you can also setup other connection from
the console using external screen and keyboard."

Step 2: Flash the SD card

All unsaved data will be lost (backup was not correct in this context)

but it's way easier to enter your passphrase this way

### CODE REVIEW

**Comment by bmegli** [comment]
And I am sorry for writing about those issues in unrelated PR (earlier).

Sometimes my fingers are faster than brain.

**Comment by bmegli** [comment]
Oh, I see what I have to fix ;-)

**Comment by bmegli** [comment]
Ok, don't merge it yet. Some links are not correct after chaning to "Flash the SD card" but I have no more time today

**Comment by bmegli** [comment]
Ok, now it's all fixed.

I didn't even know one could link this way ;-)
This is better than linking to section by id (what I do).

I am not sure if you want those changes. Just close this PR if not, I am ok with that.

**Comment by WasabiFan** [comment]
This mostly looks good. Can you add a sentence in the "Flash the SD card" section saying explicitly "back up your data or it will be destroyed"? Those two key words are important to have here.

**Comment by bmegli** [comment]
@WasabiFan 

We already have there:

"This will erase everything on your SD card! Backup your files if you do not want to lose them!"

In a red frame with exclamation mark so I think we have it covered :-)

File: docs/getting-started.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ First, here are the things you need before starting:
   * USB Ethernet (wired) dongle
   * Bluetooth
 
-  For Raspberry Pi you will need to use the wired Ethernet connection first.
-  With PiStorms the EV3 options will work too since you have a screen to configure
-  them on.
+  For RasberryPi you can use wired Ethernet connection first.
```
Comment (WasabiFan) [inline docs/getting-started.md]: This sounds a bit weird -- maybe change it to be "For Rasberry Pi, you can use a wired Ethernet connection."

File: docs/getting-started.md [hunk 2]
```diff
@@ -30,9 +30,9 @@ First, here are the things you need before starting:
...
+  With PiStorms the EV3 options will work too since you have a built-in screen to configure them on.
+  With BrickPi you can also setup other connection from the console using external screen and keyboard.
```
Comment (WasabiFan) [inline docs/getting-started.md]: This is completely subjective, but generally I think of "setup" as a noun (e.g. "my setup") and "set up" as a verb form ("I set up my EV3"). I'd rather have it changed here to "set up".

Comment (WasabiFan) [inline docs/getting-started.md]: Plural "connections"

Comment (WasabiFan) [inline docs/getting-started.md]: "an" between "using" and "external"

**Comment by WasabiFan** [comment]
I really shouldn't review code at 5:00 in the morning :laughing: I meant in the location where you made the other change, but honestly I think I'm happy without it. I just added a review with minor fix suggestions.

**Comment by bmegli** [comment]
😆 3 fixes in one sentence. I think even my teachers weren't so demanding ;-)

But yes, you're right! I'll fix it.

**Comment by WasabiFan** [comment]
It's a page that hundreds of people a day read, so I take extra care in trying to make it clear :laughing: But yeah that's a much higher standard than I usually use.

**Comment by bmegli** [comment]
Hey, I haven't had time to fix those last things ;-)

Come on, it's like teacher grabbing the exam sheet of paper from your hands...

> It's a page that hundreds of people a day read

Hundreds? Hundreds of thousands... -)

File: docs/getting-started.md (hunk 2 above)
Comment (bmegli) [inline docs/getting-started.md]: I can change it but we use setup as verb all over the page so I would than change it all over.

You can setup additional connections using the
`connmanctl` command.

 Step 5: Setup a network connection

If that's desired.

Comment (bmegli) [inline docs/getting-started.md]: I can't defend this one ;-)

 However... why would you need connections, connection is enough... ;-)

Comment (bmegli) [inline docs/getting-started.md]: articles... we non-natives never really get them right.

**Comment by WasabiFan** [comment]
Sorry... I'm really distracted today. Can you open another PR with those small fixes?

**Comment by bmegli** [comment]
No problem. You actually did titanic work with restructuring the whole page over the last week.

**Comment by bmegli** [comment]
Ok, it's in #265

**Pull request merged**